docs/libraries.md

```diff
@@ -1,52 +1,83 @@
 ---
 
-* Table of Contents
-{:toc}
+Before you can start writing code that runs on ev3dev, you need to choose a programming language. Community members have built libraries that let you use features
+such as motors and sensors from your code. Choose the language you want from below and follow the link to the library's homepage get started.
 
-If you are looking to write a program that takes advantage of the EV3's motors, sensors, or other native devices,
-using a language binding is the way to go.
-We have a repository of officially maintained language bindings, as well as many more that are maintained separately by contributors on GitHub.
+## Python
+<https://github.com/rhempel/ev3dev-lang-python>
 
-## Unified Libraries
+Python is a high-level, general-purpose, interpreted scripting language. It is great for beginners, while also powerful for experienced coders.
+If you are new to programming and unsure which language to choose, Python is a great choice!
 
-Our official "unified" [language binding repository](http://github.com/ev3dev/ev3dev-lang)
-supports multiple implementations of the ev3dev API in a variety of languages.
+**Warning!** There is another Python library created by @topikachu which is _not_ the same as this one. When searching for documentation online, make sure that you are reading about @rhempel's `ev3dev-lang-python`.
 
-* Unified bindings:
-    * [C++](https://github.com/ddemidov/ev3dev-lang-cpp)
-    * [Node.js](https://github.com/wasabifan/ev3dev-lang-js)
-    * [Python](https://github.com/rhempel/ev3dev-lang-python)
-    * [Java](https://github.com/mob41/ev3dev-lang-java)
+[Get started with Python](https://github.com/rhempel/ev3dev-lang-python){: .btn .btn-default }
 
-## Programming toolkit for ev3dev
+## JavaScript
+<https://github.com/wasabifan/ev3dev-lang-js>
 
-There is also a [GLib]/[GObject] based [toolkit][ev3devKit] that provides a number of
-programming interfaces for ev3dev, including user interface and device driver
-interface.  It is written in vala, but since it uses GObjects, it can be used
-with many [languages] via [GObjectIntrospection]. The [Brick Manager][brickman]
+JavaScript is the scripting language of the Web. With a program called Node.js, you can write JavaScript that can be run locally like any other scripting language.
+JavaScript with Node.js is great for writing web servers and other asynchronous programs which don't need to run tight loops or precise timers. It has a syntax similar to that of C and related
+languages, and is relatively easy to learn.
+
+[Get started with JavaScript](https://github.com/wasabifan/ev3dev-lang-js){: .btn .btn-default }
+
+## Go
+<https://github.com/ev3go/ev3dev>
+
+Go is a compiled, statically-typed language created by at Google. It aims to be simple and light while still providing modern language features.
```

Comment:
Another awesome advantage of go is that it has a built-in cross compiler, so even though this is a compiled language, you don't have to mess with setting up our Docker image or another cross compiler to get code that will run on the EV3.